src/QueryParameterValidator/QueryParameterValidator.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\QueryParameterValidator;
+
+use ApiPlatform\Api\FilterLocatorTrait;
+use ApiPlatform\Exception\FilterValidationException;
```

Comment:
@soyuka This is not the right exception, the scope is not binded to the filter but to the query ? how to deal with it ?